src/Raven.Studio/typescript/viewmodels/resources/widgets/settings/storageWidgetSettings.ts

```diff
@@ -0,0 +1,26 @@
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import storageKeyProvider = require("common/storage/storageKeyProvider");
+
+class storageWidgetSettings extends dialogViewModelBase {
+
+    static localStorageName = storageKeyProvider.storageKeyFor("storageWidgetScaleSettings");
```

Comment:
I think we don't have to do that manually, see: https://github.com/ravendb/ravendb/blob/9003da4c0dd23731c8f10d2649543bbfcf206503/src/Raven.Studio/typescript/viewmodels/resources/widgets/widget.ts#L110

Reply:
done - now using the widget configuration
but there is one overhead:
I needed to trigger writing to the local storage upon clicking 'save' in the settings dialog

Reply:
not a problem - we can expose method which says: any widget configratuion was updated - do save to local storage for me.

Reply:
IIRC we currently save with debounce few seconds after user finish dealing with layout. After configuration change we can save immediately.